rectangle_form.py

```diff
@@ -4,41 +4,54 @@
 from abcdforminterface import FormInterface
 
 class RectangleForm(FormInterface):
-    def __init__(self, x1, y1, width, height, color=QtGui.QColor(0, 0, 0)):
+    def __init__(self, x1, y1, width, height, location, color=QtGui.QColor(0, 0, 0)):
```

Comment:
beware with mutability issue in kwargs

def f(a=[])
  a.append('tata')
  print a


f()
f()

> tata
> tata, tata